entries/jQuery.speed.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<entry type="method" name="jQuery.speed" return="PlainObject">
+  <title>jQuery.speed</title>
+  <desc>Creates an object containing a set of properties ready to be used in the definition of custom animations.</desc>
+  <signature>
```

Comment:
There are multiple signatures. You can pass an object instead of individual parameters.